packages/apps/plugins/plugin-space/src/components/SpacePresence.tsx

```diff
@@ -90,59 +108,62 @@ export type MemberPresenceProps = ThemedClassName<{
-  const { t } = useTranslation(SPACE_PLUGIN);
+  const { members = [], size = 9, onMemberClick } = props;
 
   if (members.length === 0) {
     return null;
   }
 
   return (
     <AvatarGroup.Root size={size} classNames='mbs-2 mie-4'>
-      {members.slice(0, 3).map((member, i) => {
-        const memberHex = member.identity.identityKey.toHex();
-        const status = member.match ? 'current' : 'active';
-        const name = member.identity.profile?.displayName ?? generateName(member.identity.identityKey.toHex());
-        return (
-          <Tooltip.Root key={memberHex}>
-            <Tooltip.Trigger>
-              <AvatarGroupItem.Root color={getColorForValue({ value: memberHex, type: 'color' })} status={status}>
-                <Avatar.Frame style={{ zIndex: members.length - i }} onClick={() => onMemberClick?.(member)}>
-                  <Avatar.Fallback text={name[0]} />
-                </Avatar.Frame>
-              </AvatarGroupItem.Root>
-            </Tooltip.Trigger>
-            <Tooltip.Portal>
-              <Tooltip.Content side='bottom'>
-                <span>{name}</span>
-                <Tooltip.Arrow />
-              </Tooltip.Content>
-            </Tooltip.Portal>
-          </Tooltip.Root>
-        );
-      })}
+      {members.slice(0, 3).map((member, i) => (
+        <Tooltip.Root key={member.identity.identityKey.toHex()}>
+          <Tooltip.Trigger>
+            <PrensenceAvatar
+              identity={member.identity}
+              group
+              match={member.match}
+              index={members.length - i}
+              onClick={() => onMemberClick?.(member)}
+            />
+          </Tooltip.Trigger>
+          <Tooltip.Portal>
+            <Tooltip.Content side='bottom'>
+              <span>{getName(member.identity)}</span>
+              <Tooltip.Arrow />
+            </Tooltip.Content>
+          </Tooltip.Portal>
+        </Tooltip.Root>
+      ))}
 
       {members.length > 3 && (
         <Tooltip.Root>
           <Tooltip.Trigger>
             <AvatarGroupItem.Root color='#ccc' status='inactive'>
-              <Avatar.Frame style={{ zIndex: members.length - 4 }} onClick={onMoreClick}>
+              <Avatar.Frame style={{ zIndex: members.length - 4 }}>
                 {/* TODO(wittjosiah): Make text fit. */}
                 <Avatar.Fallback text={`+${members.length - 3}`} />
               </Avatar.Frame>
             </AvatarGroupItem.Root>
           </Tooltip.Trigger>
           <Tooltip.Portal>
             <Tooltip.Content side='bottom'>
-              <span>
-                {t('viewers label', { count: members.filter((member) => member.match).length })}
-                {t('members online label', { count: members.length - members.filter((member) => member.match).length })}
-              </span>
               <Tooltip.Arrow />
+              <List classNames='max-h-56 overflow-scroll flex flex-col gap-2'>
```

Comment:
`overflow-y-auto` is preferred, the `scroll` value forces a scrollbar to appear.

Reply:
`flex flex-col` is almost always unnecessary — I believe `List` has that by default anyway, but even that shouldn’t apply that since `block` (the default) is better to work with wherever possible.